app/components/general/components/bc-table/macro.njk

```diff
@@ -73,7 +73,8 @@
               {{ viewDataText({
                 dataTestId: dataPoint.dataTestId,
                 classes: dataPoint.classes,
-                data: dataPoint.data
+                data: dataPoint.data,
+                dataOrderItemId: dataPoint.dataOrderItemId if dataPoint.dataOrderItemId
```

Comment:
is the `if` needed? wont it be undefined either-way if it dataPoint doesnt have a dataOrderItemId property?

Reply:
Without the if statement the template fails to render when the property is not present, on list orders page